Restore and update Payment Services dropin docs (#852)
commerce-docs/microsite-commerce-storefront: src/content/docs/dropins/payment-services/containers/index.mdx

| Container | Description |
| --------- | ----------- |

> | [ApplePay](/dropins/payment-services/containers/apple-pay/) | The `ApplePay` container renders a button that macOS/iOS users can use to check out using Apple Pay. |

Reviewer: ```suggestion
| [ApplePay](/dropins/payment-services/containers/apple-pay/) | The `ApplePay` container renders a checkout button that enables macOS and iOS users to pay using Apple Pay. |
```

Author: Same; as `ApplePay` refers to the react component, should we keep the backticks? If not, we should remove the backticks also from `CreditCard`.

Reviewer: Keep. Made the updates.

> | [CreditCard](/dropins/payment-services/containers/credit-card/) | The `CreditCard` container renders a credit card form that shoppers can use to enter credit card details during the checkout process. |

Reviewer: ```suggestion
| [CreditCard](/dropins/payment-services/containers/credit-card/) | The `CreditCard` container renders a form for shoppers to enter credit card details during checkout. |
```
